lib/PuppeteerSharp/FrameManager.cs

```diff
@@ -13,7 +13,7 @@ namespace PuppeteerSharp
 {
     internal class FrameManager
     {
-        private Dictionary<int, ExecutionContext> _contextIdToContext;
+        private ConcurrentDictionary<int, ExecutionContext> _contextIdToContext;
```

Comment:
After testing my changes, I found races when accessing this dictionary, so I replaced with `ConcurrentDictionary`.